### PR TITLE
do not gather facts for availability check

### PR DESCRIPTION
**Pull request by fatz**

### CODE REVIEW

**Review by geekbass — DISMISSED** [review]
LGTM thx

File: dcos.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 - name: Wait for instances to become reachable
   hosts: all
+  gather_facts: no
   tasks:
     - name: Wait for instances to become reachable
       wait_for_connection:
```
Comment (bernadinm) [inline dcos.yml:7]: the line below this which contains `sleep: 10` needs to be updated to `delay: 10`

**Review by bernadinm — DISMISSED** [review]

File: dcos.yml (hunk 1 above)
Comment (sebbrandt87) [reply dcos.yml:7]: an additional `sleep` should be kept as well I would say.

**Review by sebbrandt87 — APPROVED** [review]

**Review by geekbass — APPROVED** [review]
LGTM

**Pull request merged**